### PR TITLE
feat: LibreOffice Calc

### LINKED ISSUE #40: Libreoffice Calc
Would we be able to install Libreoffice Calc on the desktop image? I'm thinking that a Spreadsheet program could be helpful as analysts may bee to wrangle Excel files.

I think there's also some merit to having Libreoffice Writer, but Calc is probably most important.

CC @brendangadd @frazs

### PR DESCRIPTION
**Pull request by frazs**
Resolves https://github.com/StatCan/kubeflow-containers-desktop/issues/40

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -676,7 +676,9 @@ RUN \
 ### END CONFIGURATION ### 
 
 # Small baked-in tool requests
-RUN /bin/bash $RESOURCES_PATH/tools/emacs.sh && \
+RUN add-apt-repository ppa:libreoffice/ppa && \
```
Comment (brendangadd) [inline base/Dockerfile:679]: I'm used to LibreOffice being the core repositories. Why the separate PPA? Required?

File: base/Dockerfile [hunk 2]
```diff
@@ -676,7 +676,9 @@ RUN \
...
+    apt-get install libreoffice && \
```
Comment (brendangadd) [inline base/Dockerfile]: Let's just install Calc.

File: base/Dockerfile [hunk 3]
```diff
@@ -699,6 +701,9 @@ RUN cp /usr/share/applications/firefox.desktop $HOME/Desktop/firefox.desktop &&
     echo "[Desktop Entry]\nVersion=1.0\nType=Link\nName=Netdata\nComment=Hardware Monitoring\nCategories=System;Utility;Development;\nIcon=/resources/icons/netdata-icon.png\nURL=http://localhost:8092/tools/netdata" > $HOME/Desktop/netdata.desktop && \
     chown ${NB_USER}:${NB_USER} $HOME/Desktop/netdata.desktop && \
     chmod +x $HOME/Desktop/netdata.desktop && \
+    echo "[Desktop Entry]\nVersion=1.0\nType=Application\nName=LibreOffice\nComment=The office productivity suite compatible to the open and standardized ODF document format. Supported by The Document Foundation.\nExec=libreoffice %U\nIcon=libreoffice-startcenter\nPath=\nTerminal=false\nStartupNotify=true" > $HOME/Desktop/libreoffice.desktop && \
+    chown ${NB_USER}:${NB_USER} $HOME/Desktop/libreoffice.desktop && \
+    chmod +x $HOME/Desktop/libreoffice.desktop && \
```
Comment (brendangadd) [inline base/Dockerfile]: As per above, just Calc.

**Review by brendangadd — CHANGES_REQUESTED** [review]

File: base/Dockerfile (hunk 1 above)
Comment (frazs) [reply base/Dockerfile:679]: Depends on what version we want. Separate PPA for 6.4.6.2 (the latest stable version), while the core repositories are at 6.0.7.3. Alternatively, there's a 7.0 available as a collection of debs, but I think that might still be a preview.

**Comment by frazs** [comment]
No obvious issues dropping down to just libreoffice-calc, so this can be finished as soon as we determine which version to go with. Unless there's an issue with adding a PPA, I'd lean towards the 6.4 as it includes a lot of bugfixes over 6.0 as well as a significantly smoother UI (IMO):

6.0 (in core): 
image

6.4 (in libreoffice/ppa):
image

7.0 was only released about two weeks ago, so we may want to wait until it goes through more testing or a stable release. I haven't tried it out (the installation process is more manual) but can give it a shot if desired.

**Comment by brendangadd** [comment]
@frazs Avoiding additional PPAs is desirable but, after a bit of research, I think your course of action (adding it) is best. Also, agreed on the clean UI!

Latest stable it is.

**Comment by frazs** [comment]
Ah, I was wrong on the UI issue. It is not a difference of 6.0 vs 6.4, but a difference of libreoffice-calc alone vs as part of libreoffice. Not sure what the broader libreoffice package is bringing in to change that...

**Comment by frazs** [comment]
Sorted it: the missing UI package in the standalone install was `libreoffice-gtk3`.

**Review by brendangadd — APPROVED** [review]

**Pull request merged**